models/staging/tpdm/base/base_tpdm__education_organizations.sql

```diff
@@ -0,0 +1,31 @@
+with education_organizations as (
+    {{ source_edfi3('education_organizations') }}
+),
+renamed as (
```

Comment:
could we add a newline between CTEs?
```suggestion
),

renamed as (
```